src/commands/moderation/prefixCommands/functions/deleteContent.ts

```diff
@@ -0,0 +1,145 @@
+});
+
+const noContentEmbed = (command: string, version: string) => makeEmbed({
+    title: 'Prefix Commands - Delete Content - No Content',
+    description: `Failed to delete command content for command ${command} and version ${version} as the content does not exist.`,
+    color: Colors.Red,
+});
+
+const noCommandEmbed = (command: string) => makeEmbed({
+    title: 'Prefix Commands - Delete Content - No Command',
+    description: `Failed to delete command content for command ${command} as the command does not exist or there are more than one matching.`,
+    color: Colors.Red,
+});
+
+const noVersionEmbed = (version: string) => makeEmbed({
+    title: 'Prefix Commands - Delete Content - No Version',
+    description: `Failed to delete command content for version ${version} as the version does not exist or there are more than one matching.`,
+    color: Colors.Red,
+});
+
+const failedEmbed = (version: string) => makeEmbed({
+    title: 'Prefix Commands - Delete Content - Failed',
+    description: `Failed to delete the prefix command content with version ${version}.`,
+    color: Colors.Red,
+});
+
+const successEmbed = (command: string, version: string) => makeEmbed({
+    title: `Prefix command content for command ${command} and version ${version} was deleted successfully.`,
+    color: Colors.Green,
+});
+
+const modLogEmbed = (moderator: User, commandName: string, versionName: string, title: string, content: string, image: string) => makeEmbed({
+    title: 'Prefix command content delete',
+    fields: [
+        {
+            name: 'Command',
+            value: commandName,
+        },
+        {
+            name: 'Version',
+            value: versionName,
+        },
+        {
+            name: 'Title',
+            value: title,
+        },
+        {
+            name: 'Content',
+            value: content,
+        },
+        {
+            name: 'Image',
+            value: image,
+        },
+        {
+            name: 'Moderator',
+            value: `${moderator}`,
+        },
+    ],
+    color: Colors.Red,
+});
+
+const noModLogs = makeEmbed({
+    title: 'Prefix Commands - Delete Content - No Mod Log',
+    description: 'I can\'t find the mod logs channel. Please check the channel still exists.',
+    color: Colors.Red,
+});
+
+export async function handleDeletePrefixCommandContent(interaction: ChatInputCommandInteraction<'cached'>) {
+    await interaction.deferReply({ ephemeral: true });
+
+    const conn = getConn();
+    if (!conn) {
+        await interaction.followUp({ embeds: [noConnEmbed], ephemeral: true });
+        return;
+    }
+
+    const command = interaction.options.getString('command')!;
+    const version = interaction.options.getString('version')!;
+    const moderator = interaction.user;
+
+    //Check if the mod logs channel exists
+    const modLogsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_LOGS) as TextChannel;
+    if (!modLogsChannel) {
```

Comment:
```suggestion
    let modLogsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_LOGS);
    if (!modLogsChannel || !modLogsChannel.isTextBased()) {
        modLogsChannel = null;
```
Might be safer to actually check if the channel is a text channel? Especially if we want to offer the codebase to other projects. Then, if it's not a text channel set it to null to prevent accidental attempts to send into a non-text channel.